### PR TITLE
Update FAQ and Minecraft Compatibility

### PR DESCRIPTION
**Pull request by jackjt8**
Update FAQ with information on WIP projects like ChunkyCL (GPU rendering) and the json-block-models branch/plugin (mod blocks and custom block models).

image

Updated Minecraft Compatibility with a table of unsupported stuff for both 1.4.5 and 2.3.0.

image

(background is broken for local builds)

### CODE REVIEW

File: style/style.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 
 tr:nth-child(even) {
-  background: #E3E3E3;
+  background: #e3e3e32e;
```
Comment (leMaik) [inline style/style.css:4]: This won't work in IE11 but… I guess that doesn't matter

Comment (jackjt8) [reply style/style.css:4]: Oh wait

**Review by leMaik — APPROVED** [review]
The table styles could use some refactoring but I mostly care about the content so… :+1:

**Comment by jackjt8** [comment]
I forgot to apply the fixed rgb value.

File: docs/faq.md [hunk 2]
```diff
@@ -19,20 +19,19 @@ Please post bugs on the [chunky GitHub page.][1]
...
-  emitters to speed up the convergence rate. See the [Path Tracing](/path_tracing.html) article for
-  more details.
+  days to render a nice image. You can reduce the size of the canvas, disable
+  emitters, enable ESS, or use a denoising technique to speed up the convergence rate. See the [Path Tracing](/path_tracing.html) article or [jackjt8's Guide to Chunky - Denoising][5] for more details.
 
 * **Q: Is GPU rendering supported?**
-  GPU support is not actively being worked on right now. GPU rendering *may* be
-  added in the future, and some partial progress has been made toward this goal
-  but there are very many hurdles before it is fully supported.
+  GPU rendering support for Chunky 2.3 is currently in development in the form of an OpenCL 1.2 renderer plugin. This renderer is still under development and many features of the CPU renderer are not yet supported. For more information and WIP builds please visit the plugins GitHub - [ChunkyCL][6].
```
Comment (leMaik) [inline docs/faq.md:26]:
```suggestion
  GPU rendering support for Chunky 2.3 is currently in development in the form of an OpenCL 1.2 renderer plugin. This renderer is still under development and many features of the CPU renderer are not yet supported. For more information and WIP builds please visit [the plugins GitHub][6].
```

**Review by leMaik — APPROVED** [review]

**Pull request merged**